app/services/instagram/send_on_instagram_service.rb

```diff
@@ -51,7 +51,6 @@ def attachament_message_params
   def send_to_facebook_page(message_content)
     access_token = channel.page_access_token
     app_secret_proof = calculate_app_secret_proof(GlobalConfigService.load('FB_APP_SECRET', ''), access_token)
-
```

Comment:
intended?

Reply:
Just noticed that this line was not needed.